aten/src/ATen/native/quantized/cpu/qbatch_norm.cpp

```diff
@@ -274,7 +274,7 @@ class QBatchNorm3d final : public torch::OperatorKernel {
 };
 
 static auto registry = torch::RegisterOperators().op(
-    "quantized::batch_norm(Tensor qx, "
+    "quantized::batch_norm2d(Tensor qx, "
```

Comment:
do you need to keep `quantized::batch_norm` for BC?

Reply:
I don't think so. The `aten::quantized_batch_norm` is still preserving bc.
cc @lly-zero-one for new op signature.

Reply:
Can you also change the convert script?

Reply:
NVM, I think I just need to update the model.